requirements.txt

```diff
@@ -2,4 +2,4 @@ coverage==3.7.1
 flake8==2.1.0
 mock==1.0.1
 nose==1.3.0
-requests-futures==0.9.4
+requests-futures>=0.9.9
```

Comment:
Please change this dependency also in setup.py

Reply:
waiting for this merge, any update?